index.js

```diff
@@ -16,12 +16,27 @@ const digit = numbers => {
   return mod < 2 ? 0 : 11 - mod
 }
 
+/**
+ * To clean the CNPJ
+ * @param {String|Number} cnpj 
+ */
+export function cleanse(cnpj) {
+  if(!validate(cnpj)) {
+    return cnpj
+  }
+  return cnpj.toString().replace(/\D/g,'');
+}
+
 /**
  * Validates a CNPJ
  * @param {String|Number} cnpj The CNPJ value to be validated
  * @return {Boolean}
  */
 export function validate(cnpj) {
+  // Verify reference
+  if(!cnpj) {
+    return false
+  }
```

Comment:
It's unnecessary to check this here, since it's being handled by line 45. Please remove.